Exercise 20-JS/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+let age="100";
+console.log(Number(age));
+
+let firstName=true;
+firstName = String("Carlos");
+console.log(firstName);
+
+let a = Boolean('You have completed this course yet');
```

Comment:
What this code does is trying to convert the text "You have completed this course yet" to a boolean. It should be a variable as per the following;
`let courseCompleted = false;`